L5_Application/VS1053.cpp

```diff
@@ -105,66 +103,21 @@ void VS1053::sineTest(uint8_t n, uint16_t ms) {
   vTaskDelay(500);
 }
 
-void VS1053::setVolume(uint8_t left, uint8_t right) {
-  uint16_t volume;
-  volume = left;
-  volume <<= 8;
-  volume |= right;
-  sciWrite(SCI_VOL, volume);
+void VS1053::sendVolume(uint8_t left, uint8_t right) {
+  uint16_t v;
+  v = left;
+  v <<= 8;
+  v |= right;
+  sciWrite(SCI_VOL, v);
+}
+void VS1053::setVolume(uint8_t v)
```

Comment:
Poor variable naming, please change before pr is accepted

Reply:
Changes added in bdf5957